docs/streams/upgrade-guide.html

```diff
@@ -849,13 +849,13 @@ <h3><a id="streams_api_changes_270" href="#streams_api_changes_270">Streams API
         <code>KeyQueryMetadata</code> was introduced in Kafka Streams 2.5 release with getter methods having prefix <code>get</code>.
         The intend of this change is to bring the method names to Kafka custom to not use the <code>get</code> prefix for getter methods.
         The old methods are deprecated and is not effected.
-        (Cf. <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-648%3A+Renaming+getter+method+for+Interactive+Queries">KIP-648</a>.)
+        (Cf. <a href="https://cwiki.apache.org/confluence/x/vyd4CQ">KIP-648</a>.)
     </p>
     <p>
         The <code>StreamsConfig</code> variable for configuration parameter <code>"topology.optimization"</code>
         is renamed from <code>TOPOLOGY_OPTIMIZATION</code> to <code>TOPOLOGY_OPTIMIZATION_CONFIG</code>.
         The old variable is deprecated. Note, that the parameter name itself is not affected.
-        (Cf. <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-626%3A+Rename+StreamsConfig+config+variable+name">KIP-629</a>.)
+        (Cf. <a href="https://cwiki.apache.org/confluence/x/gBB4CQ">KIP-629</a>.)
```

Comment:
This link is redirected to KIP-626.
According to the context, I think it might be wrong for the KIP number, not the hyperlink.

Reply:
I think it should be KIP-626 instead, as KIP-629 seems to be incorrect.

- [KIP-629: Use racially neutral terms in our codebase](https://cwiki.apache.org/confluence/display/KAFKA/KIP-629%3A+Use+racially+neutral+terms+in+our+codebase)
- [KIP-626: Rename StreamsConfig config variable name](https://cwiki.apache.org/confluence/display/KAFKA/KIP-626%3A+Rename+StreamsConfig+config+variable+name)